plugins/bundle/status.go

```diff
@@ -47,6 +48,10 @@ func (s *Status) SetActivateSuccess(revision string) {
 // download.
 func (s *Status) SetDownloadSuccess() {
 	s.LastSuccessfulDownload = time.Now().UTC()
+
+	if s.FirstSuccessfulActivation.IsZero() {
+		s.FirstSuccessfulActivation = s.LastSuccessfulDownload
```

Comment:
Why are we setting the `FirstSuccessfulActivation` to last `LastSuccessfulDownload`. A successful download does not imply a successful activation. I could imagine setting this value in the `SetActivateSuccess` method.

Reply:
Thanks, @ashutosh-narkar. I missed that we already keep the last success bundle activation. 
In light of this, I changed the code a bit:
1. we keep the activeRevision label (just on) the last_success_bundle_activation metric.
2. the gauge gets reset, so we only keep the last active_revision (instead of keeping them all and therefore avoiding the situation where the /metrics output grows indefinitely)

What do you think, @rafaelreinert?

```
# HELP last_success_bundle_activation Gauge for the last success bundle activation.
# TYPE last_success_bundle_activation gauge
last_success_bundle_activation{active_revision="1650490742504",name="authz"} 1.650490748253708e+18
# HELP last_bundle_request Gauge for the last bundle request.
# TYPE last_bundle_request gauge
last_bundle_request{name="authz"} 1.650490762751247e+18
# HELP last_success_bundle_download Gauge for the last success bundle download.
# TYPE last_success_bundle_download gauge
last_success_bundle_download{name="authz"} 1.6504907480146977e+18
# HELP last_success_bundle_request Gauge for the last success bundle request.
# TYPE last_success_bundle_request gauge
last_success_bundle_request{name="authz"} 1.650490762751247e+18
# HELP plugin_status_gauge Gauge for the plugin by status.
# TYPE plugin_status_gauge gauge
plugin_status_gauge{name="bundle",status="OK"} 1
plugin_status_gauge{name="decision_logs",status="OK"} 1
plugin_status_gauge{name="discovery",status="OK"} 1
plugin_status_gauge{name="status",status="OK"} 1
```

I also managed to use the `last_success_bundle_activation` to add an annotation in Grafana, which contains the `active_revision` label. In this way, we are able to see when (and which) new bundle is activated in OPA and correlate with the other metrics.

<img width="1238" alt="Screenshot 2022-04-21 at 00 48 25" src="https://user-images.githubusercontent.com/952836/164328831-c339a69f-25a0-4945-bc27-5d73faeda00b.png">

<img width="1274" alt="Screenshot 2022-04-21 at 00 41 00" src="https://user-images.githubusercontent.com/952836/164328490-0cbfec2e-4e7c-417e-bb46-3c90350a52f4.png">

Reply:
LGTM, solves the cardinality issue and keeps the active_revision of the last bundle. 😄